data/zilliqa/assets/zil1pqcev4ykxla0jhy3anx32lnqgv8xncd8q57ql2/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Sparda Wallet",
+    "symbol": "SPW",
+    "web": "https://spardawallet.com",
+    "supply": "100000000",
+    "email": "spardawallet@gmail.com",
+    "whitepaper": "https://drive.google.com/file/d/1nuJGF_OeAEtZDtyQzaBSoKQkClkFObRt/view?usp=sharing",
+    "publicTeam": true,
```

Comment:
remove, the ones listed from the WP are nowhere near enough of proof

Reply:
Can you please explain, in more detail? remove what? and by "ones" what exactly do you mean?

Reply:
Remove the public team property

Reply:
I am going to assume the public team, considering we are only 2 people.